### DISCUSSION TITLE
jakarta.mail.AuthenticationFailedException with valid credentials

### DISCUSSION THREAD

**Discussion by Tasza33** [qa-guru/allure-notifications#264] [Q&A]
Hi community. I'm trying to send email without SSL, my config looks like:

```
\"mail\": {
        \"host\": \"mail.com\",
        \"port\": \"25\",
        \"username\": \"svc..@mail.com\",
        \"password\": \"${CI_EMAIL_PASSWORD}\",
        \"securityProtocol\": \"null\",
        \"from\": \"svc..@mail.com\",
        \"recipient\": \"n...@mail.com\"
  } 
```

Credentials are valid, that was tested. Could you help me?
Full log [java.log](https://github.com/qa-guru/allure-notifications/files/14873304/java.log)

Used image [dquintela/openjdk-8-jdk-alpine](https://hub.docker.com/layers/dquintela/openjdk-8-jdk-alpine/latest/images/sha256-612698b405dbd06bfc449d12677aa10ce4625425ed2a6ccfab401ee2d6ac8d38)

**Comment by valfirst** [comment]
You need to check the restrictions of your mailing provider. There could be a number of reasons, e.g. MFA or some settings disabled.
